src/components/ZetkinPerson.tsx

```diff
@@ -14,9 +14,7 @@ const ZetkinPerson: React.FunctionComponent<{
             <Avatar src={ orgId ? `/api/orgs/${orgId}/people/${id}/avatar` : '' } />
             <Box alignItems="start" display="flex" flexDirection="column" justifyContent="center" ml={ 1 }>
                 <Typography variant="body1">{ name }</Typography>
-                { subtitle && (
-                    <Typography variant="body2">{ subtitle }</Typography>
-                ) }
+                { typeof subtitle === 'string' ? <Typography>{ subtitle }</Typography> : <Box alignItems="center" display="flex">{ subtitle }</Box> }
```

Comment:
The line is very long, how could we break it up?

Also, do we need the `<Box>` here, or could it be up to the user of the component to create a box if necessary?